sdpoker.js

```diff
@@ -33,10 +33,11 @@ const args = yargs
   .default('multicast', false)
   .default('unicast', false)
   .default('shaping', false)
+  .default('verbose', false)
   .boolean([ 'checkEndings', 'should', 'noCopy', 'noMedia',
     'duplicate', 'videoOnly', 'audioOnly', 'channelOrder',
-    'useIP4', 'useIP6', 'multicast', 'unicast', 'shaping' ])
-  .usage('Check an SDP file for conformance with RFC4566 and SMPTE ST 2110.\n' +
+    'useIP4', 'useIP6', 'multicast', 'unicast', 'shaping', 'smpte2110_22', 'verbose' ])
+  .usage('Check an SDP file for conformance with RFC4566, SMPTE ST 2110 and SMPTE ST 2110-22.\n' +
```

Comment:
`SMPTE ST 2110` encompasses `ST 2110-22` I think...

```suggestion
  .usage('Check an SDP file for conformance with RFC4566 and SMPTE ST 2110.\n' +
```